receiver/vcenterreceiver/scraper.go

```diff
@@ -399,13 +400,25 @@ func (v *vcenterMetricScraper) collectVMs(
 			continue
 		}
 
-		switch vm.Runtime.PowerState {
-		case "poweredOff":
-			poweredOffVMs++
-		case "poweredOn":
-			poweredOnVMs++
-		default:
-			suspendedVMs++
+		if vm.Config.Template {
+			templates++
+		} else {
+			switch vm.Runtime.PowerState {
+			case "poweredOff":
+				poweredOffVMs++
+			case "poweredOn":
+				poweredOnVMs++
+			default:
+				suspendedVMs++
+			}
+		}
+
+		// TODO: Remove after v0.100.0 has been released
```

Comment:
well right now, I believe that if I didn't have this code, the VM template resource would still get created if these attributes were disabled...but it would simply consist of an

- Optionally enabled datacenter attribute
- Possible cluster attribute
- Host attribute

This would not be unique enough and not make any sense.

Reply:
Would it make sense then to only `continue` if both resource attributes are disabled then? If one or the other is enabled, wouldn't the user want that information?

Reply:
@crobert-1 actually, I think that probably makes enough sense. I can make this change.